pyconbalkan/organizers/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+
+from pyconbalkan.organizers.models import Volunteer
+
+
+class VolunteerCreateForm(forms.ModelForm):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        # delete labels, add form-control class to all fields
+        for f in self.fields.values():
+            old_classes = f.widget.attrs['class'] if 'class' in f.widget.attrs else ''
+            f.widget.attrs.update({'placeholder': f.label, 'class': f'{old_classes} form-control'})
+            f.label = ''
+
+    class Meta:
+        model = Volunteer
+        exclude = ('active', 'user', 'weight', 'type', 'slug', 'description', )
```

Comment:
You can remove `weight`  (because I remove it) and `description` (so they can fill this up). 🔝😃

Reply:
done :)